.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,8 @@
-blank_issues_enabled: true
+blank_issues_enabled: false
```

Comment:
I guess @antocuni asked specifically to be able to create empty templates for having some freedom when he was creating issues that didn't fit the other categories. Wondering if you still have opinions on this Antonio or fine with discussions?

Reply:
I think the variety of Github Discussion categories should appease Antonio (⛷️ ). Let's ship it, it is just a boolean away to re-enable.